pages/requester/manage-task/[taskid].tsx

```diff
@@ -185,25 +219,44 @@ export default function ManageResponses() {
     <>
       <PageHeader title="Requester Manage Tasks" />
       <LoadingOverlay open={openLoading} text="Processing Approval..." />
-      <PrimaryButtonCTA
-        text="← My Tasks"
-        size="small"
-        to="/requester/my-tasks"
-        sx={{ mx: "auto", mt: 2 }}
-      />
+      <Box
+        sx={{
+          display: "flex",
+          justifyContent: "center",
+        }}
+      >
+        <PrimaryButtonCTA
+          text="← My Tasks"
+          size="small"
+          to="/requester/my-tasks"
+          sx={{ mr: 1, mt: 2 }}
+        />
+        <PrimaryButtonCTA
+          text="Download Results"
+          size="small"
+          onClick={() => {
+            handleDownloadExcel(query.taskid as string);
+          }}
+          sx={{ ml: 1, mt: 2 }}
+        />
+      </Box>
       <ApprovalsTable data={data} extraColumns={extraColumns} />
     </>
   );
 }
 
 const approvalStatusMap = {
-  1: "Not reviewed",
+  1: "Not Reviewed",
```

Comment:
Let's keep these consistent across the board with the other status map:
  0: "In progress",
  1: "Pending approval",
  2: "Ready to Withdraw",
  3: "Abandoned",
  4: "Paid",
  5: "Rejected",